### DISCUSSION TITLE
Get the links to included images via script

### DISCUSSION THREAD

**Discussion by doctomlug** [freeplane/freeplane#2002] [Help] [answered]
Hi,
I want to write a script that checks which included images are not required anymore. Traversing all nodes of the map is simple enough but I am unable to find the storage location of the links. 
I searched the node via the inspection tool of the groovy console: no luck :-(

What did I overlook?
Thank you for your support.

**Comment by euu2021** [comment, marked as answer]
This: `node.getExternalObject().getUri()`

I discovered by looking at the [ImgInscpector ](https://github.com/i-plasm/freeplane-scripts/blob/main/src/scripts/freeplaneImgInspector.groovy) script.

**Comment by euu2021** [comment]
By the way, this script that you are creating is interesting. In the Anki software, for example, there is a feature called "Check Media", that allows deleting unused media files.

Please, share when you create.

**Comment by doctomlug** [comment]
Thanks, will check it out soon.  [Edit: yes, that's it. ty]
Where to share the creations?
The FreePlane web page looks kind of deserted and outdated.
Here? 
And which format? So far I have not built any addons, just a ton of more or less user(un)friendly scripts. [Edit: answered by the provided link]

**Reply by euu2021** [reply]
> [Edit: answered by the provided link]

Another example is the [macmarrun collection](https://github.com/macmarrum/freeplane-scripting/tree/main/src/scripts).
